clients/src/main/resources/common/message/StreamsGroupDescribeResponse.json

```diff
@@ -0,0 +1,167 @@
+            { "name": "InstanceId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
+              "about": "The member instance ID for static membership." },
+            { "name": "RackId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
+              "about": "The rack ID." },
+
+            { "name": "ClientId", "type": "string", "versions": "0+",
+              "about": "The client ID." },
+            { "name": "ClientHost", "type": "string", "versions": "0+",
+              "about": "The client host." },
+
+            { "name": "TopologyEpoch", "type": "int32", "versions": "0+",
+              "about": "The epoch of the topology on the client." },
+
+            { "name": "ProcessId", "type": "string", "versions": "0+",
+              "about": "Identity of the streams instance that may have multiple clients. " },
+            { "name": "UserEndpoint", "type": "Endpoint", "versions": "0+", "nullableVersions": "0+", "default": "null",
+              "about": "User-defined endpoint for Interactive Queries. Null if not defined for this client." },
+            { "name": "ClientTags", "type": "[]KeyValue", "versions": "0+",
+              "about": "Used for rack-aware assignment algorithm." },
+            { "name": "TaskOffsets", "type": "[]TaskOffset", "versions": "0+",
+              "about": "Cumulative changelog offsets for tasks." },
+            { "name": "TaskEndOffsets", "type": "[]TaskOffset", "versions": "0+",
+              "about": "Cumulative changelog end offsets for tasks." },
+
+            { "name": "Assignment", "type": "Assignment", "versions": "0+",
+              "about": "The current assignment." },
+            { "name": "TargetAssignment", "type": "Assignment", "versions": "0+",
+              "about": "The target assignment." },
+            { "name": "IsClassic", "type": "bool", "versions": "0+",
+              "about": "True for classic members that have not been upgraded yet." }
+          ]},
+        { "name": "AuthorizedOperations", "type": "int32", "versions": "0+", "default": "-2147483648",
+          "about": "32-bit bitfield to represent authorized operations for this group." }
+      ]
+    }
+  ],
+  "commonStructs": [
+    { "name": "Endpoint", "versions": "0+", "fields": [
+      { "name": "Host", "type": "string", "versions": "0+",
+        "about": "host of the endpoint" },
+      { "name": "Port", "type": "uint16", "versions": "0+",
+        "about": "port of the endpoint" }
+    ]},
+    { "name": "TaskOffset", "versions": "0+", "fields": [
+      { "name": "SubtopologyId", "type": "string", "versions": "0+",
+        "about": "The subtopology identifier." },
+      { "name": "Partition", "type": "int32", "versions": "0+",
+        "about": "The partition." },
+      { "name": "Offset", "type": "int64", "versions": "0+",
+        "about": "The offset." }
+    ]},
+    { "name": "TopicPartitions", "versions": "0+", "fields": [
```

Comment:
Since this common struct does not seems to be used, should we remove it?

Reply:
Good find!

https://github.com/apache/kafka/pull/20864